app/tests/content/test_strike_integration.py

```diff
@@ -0,0 +1,107 @@
+from datetime import timedelta
+
+from rest_framework import status
+
+import pytest
+
+from app.common.enums import AdminGroup, Groups, StrikeEnum
+from app.content.factories import StrikeFactory
+from app.content.factories.event_factory import EventFactory
+from app.content.factories.user_factory import UserFactory
+from app.util.test_utils import get_api_client
+from app.util.utils import now
+
+API_STRIKE_BASE_URL = "/strikes/"
+
+
+@pytest.fixture()
+def strike_post_data():
+    return {
+        "user_id": UserFactory().user_id,
+        "event": EventFactory().id,
+        "description": "Strike test description",
+        "strike_size": 1,
+    }
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    ("group_name", "expected_status_code"),
+    [
+        (AdminGroup.HS, status.HTTP_200_OK),
+        (AdminGroup.INDEX, status.HTTP_200_OK),
+        (AdminGroup.NOK, status.HTTP_200_OK),
+        (AdminGroup.SOSIALEN, status.HTTP_200_OK),
+        (AdminGroup.PROMO, status.HTTP_403_FORBIDDEN),
+        (Groups.TIHLDE, status.HTTP_403_FORBIDDEN),
+    ],
+)
+def test_list_strikes_as_member_of_board_or_sub_group(
+    member, group_name, expected_status_code
+):
+    """A member of HS, Index, NOK or Sosialen can list strikes"""
+    client = get_api_client(user=member, group_name=group_name)
+    url = API_STRIKE_BASE_URL
+    response = client.get(url)
```

Comment:
Unødvendig å deklarere url lik API_STRIKE_BASE_URL
```suggestion
    response = client.get(API_STRIKE_BASE_URL)
```